_data/master/navbars/usage.yml

```diff
@@ -17,8 +17,14 @@ toc:
       path: /usage/calicoctl/configure/etcd
     - title: Kubernetes API datastore options
       path: /usage/calicoctl/configure/kdd
-- title: Securing Calico's Prometheus endpoints 
-  path: /usage/policy/secure-metrics
+- title: Securing Calico communications
+  section:
+  - title: Scheduling to well-known nodes
+    path: /usage/reduce-nodes
+  - title: Configuring encryption and authentication
+    path: /usage/encrypt-comms
+  - title: Securing endpoints with Calico policy
```

Comment:
The new title for this doc isn't actually what the doc explains. Should probably change it back.

Reply:
@neiljerram  plans to expand this section further.

Reply:
Sure...but for merges it will be more convenient.

Reply:
It's just wrong - endpoints are a Calico concept discussed elsewhere in the docs. The term "endpoints" in this title doesn't mean the same thing as everywhere else in the docs

Reply:
OK I guess I don't understand the exact distinction, but "endpoints" is in the title of @neiljerram 's existing doc https://docs.projectcalico.org/master/usage/policy/secure-metrics

Reply:
As discussed yesterday, it's not the current title - it says "Prometheus endpoints" (also I wrote that doc).

We should revert the title change.